### DISCUSSION TITLE
Change colors

### DISCUSSION THREAD

**Discussion by Greginkansas** [HASwitchPlate/openHASP#415] [Q&A] [answered]
I just can not get how to change the color of the bar
 {"page":0,"id":21,"obj":"bar","x":1,"y":240,"w":479,"h":20,"value_color50":"#22f20f","val":50,"bg_color":"#f20f2d"}
I wan a green bar on a blue background what am I doing wrong?

**Comment by fvanroie** [comment, marked as answer]
>  I wan a green bar on a blue background what am I doing wrong?

You need to appy the `bg_color` properties to[ each part by adding a **suffix**](https://www.openhasp.com/0.6.3/design/styling/#parts):

- Bar (the indicatior) = suffix `10`
- Background = suffix `00` (the same as no suffix)

```json
 {"page":0,"id":21,"obj":"bar","x":1,"y":240,"w":479,"h":20,"val":50,"bg_color00":"blue","bg_color10":"green"}
 ```

**Reply by Greginkansas** [reply]
Thank you
